source/appModules/miranda32.py

```diff
@@ -1,6 +1,6 @@
 #appModules/miranda32.py
 #A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2006-2012 NVDA Contributors
+#Copyright (C) 2006-2019 NVDA Contributors, Bill Dengler

```

Comment:
Please change this as follows:

```suggestion
#Copyright (C) 2006-2019 NV Access Limited, Aleksey Sadovoy, Peter Vágner, Joseph Lee, Bill Dengler
```

Also, please start the file with a new line, containing:
`# -*- coding: UTF-8 -*-`